package/MDAnalysis/version.py

```diff
@@ -68,3 +68,4 @@
 
 #: Release of MDAnalysis as a string, using `semantic versioning`_.
 __version__ = "2.0.0-dev0"  # NOTE: keep in sync with RELEASE in setup.py
+release = False
```

Comment:
Ideally we can have this as something like `release = not 'dev0' in __version__`?

Reply:
Oops, yeah, that's what the script does anyway so this line is redundant now.